packages/block-library/src/cover/edit.native.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Icon,
+	ImageWithFocalPoint,
+	PanelBody,
+	RangeControl,
+} from '@wordpress/components';
+import {
+	InnerBlocks,
+	InspectorControls,
+	MEDIA_TYPE_IMAGE,
+	MediaPlaceholder,
+	withColors,
+} from '@wordpress/block-editor';
+import { compose, withPreferredColorScheme } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
+import { useState, useEffect } from '@wordpress/element';
+import { cover as icon } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+import { onCoverSelectMedia } from './edit.js';
+import { COVER_MIN_HEIGHT, IMAGE_BACKGROUND_TYPE, VIDEO_BACKGROUND_TYPE } from './shared';
+
+/**
+ * Constants
+ */
+const ALLOWED_MEDIA_TYPES = [ MEDIA_TYPE_IMAGE ];
+const INNER_BLOCKS_TEMPLATE = [
+	[ 'core/paragraph', {
+		align: 'center',
+		fontSize: 'large',
```

Comment:
I'm a bit concerned about this, since we still can't render or change font sizes from the app. This might lead to the user publishing something quite different from what they're seeing on mobile. This is specially worse if they add more than one paragraph, since the first one will be large by default and the second won't, but they would look the same inside the app.

Reply:
Oh I didn't think about that 😱 good catch! For now, I'm removing setting the attribute as default until we support font sizes https://github.com/WordPress/gutenberg/pull/19722/commits/5305373a6fbae766c01f6e025bcc1fd0b37808c0